npm/css-src/sass/main.scss

```diff
@@ -14,3 +14,41 @@
   .hide {
   display: none;
 }
+
```

Comment:
Now that the CSS is working, I think it's worth moving the drag-and-drop elements into their own file. Make a new file starting with an underscore and import it like we do for the progress bar styles.